src/components/Table/Table.test.tsx

```diff
@@ -85,191 +85,6 @@ describe("given data", () => {
 
   const dueDateWarning = ["dueDate"];
 
-  it("renders the snapshot correctly", () => {
```

Comment:
This snapshot was removed because each and every time it runs, the date values change, which means the snapshots always have to be updated. I tried mocking the date but that too made for a flaky test because the underlying code always relies on `new Date()` which can't be mocked.